### ISSUE TITLE
Figcaption doesn't go on top!

### ISSUE THREAD

**Issue by farangisosivand** [farangisosivand/hot-grid-dle#1]
I have problem using positions in this assignment. My figcaption doesn't go on top of my image, there is also 2 measurements in the assignment folder that I don't know how to use them.
I attached two images of my index and css code.
@thomasjbradley

**Comment by thomasjbradley** [comment]
Have you committed & pushed? I don’t see any code in the `index.html` file.

**Comment by farangisosivand** [comment]
Now the code is visible, sorry for that. 
@thomasjbradley

**Comment by thomasjbradley** [comment]
You’ll need to use `position: absolute` & `relative` to solve this problem, along with coordinates to move the elements into place.

**Comment by farangisosivand** [comment]
I used position: relative for the img-flex and position: absolute for the figcaption but it didn't do the work 
@thomasjbradley

**Comment by thomasjbradley** [comment]
Relative always goes onto a parent element, so on `<figure>`. Absolute is in the right place: did you add coordinates, like `top`, `left`, `bottom`, etc.?

**Comment by farangisosivand** [comment]
Yes. I just added the coordinates. I have the caption on top now! thanks. I have another problem now. There is still white space on top of every image in each screen. I removed the margin from the image but still doesn't work. 
@thomasjbradley

**Comment by thomasjbradley** [comment]
It’s the margin on the `<p>` tag inside the `<figcaption>`.

**Comment by farangisosivand** [comment]
Submitted. Thanks! 
@thomasjbradley
